lib/Mojolicious/Static.pm

```diff
@@ -71,7 +71,9 @@ sub is_fresh {
 
   # If-None-Match
   $etag //= $res_headers->etag // '';
-  return undef if $match && !grep { trim($_) eq $etag } split ',', $match;
+  return undef
+    if $match && !grep $_ eq $etag || "W/$_" eq $etag, map trim($_), split ',',
```

Comment:
We don't use `map`/`grep` without block.

Reply:
ok, fixed.